sdk/eventhub/azure-eventhub/azure/eventhub/_eventprocessor/in_memory_checkpoint_store.py

```diff
@@ -143,8 +149,7 @@ def claim_ownership(self, ownership_list, **kwargs):
                 result.append(ownership)
         return result
 
-    def update_checkpoint(self, checkpoint, **kwargs):
-        # type: (Dict[str, Optional[Union[str, int]]], Any) -> None
+    def update_checkpoint(self, checkpoint: str, **kwargs: Optional[Union[str, int]]) -> None:
```

Comment:
I believe this should be the following:
```suggestion
    def update_checkpoint(self, checkpoint: Dict[str, Optional[Union[str, int]]], **kwargs: Any) -> None:
```